apps/web/src/components/nav-user.test.tsx

```
@@ -0,0 +1,75 @@
+}))
+
+vi.mock('@durabull/analytics/events', () => ({
+  AnalyticsEvents: {
+    USER_SIGNED_OUT: 'USER_SIGNED_OUT',
+    THEME_CHANGED: 'THEME_CHANGED',
+  },
+}))
+
+vi.mock('@/components/ui/dropdown-menu', () => ({
+  DropdownMenu: ({ children }: { children: React.ReactNode }) => <div>{children}</div>,
+  DropdownMenuTrigger: ({ children }: { children: React.ReactNode }) => <button>{children}</button>,
```

Biome (2.4.15)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)